platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/textureview/TextureViewDebugModeActivity.java

```diff
@@ -18,19 +17,6 @@ protected MapboxMapOptions setupMapboxMapOptions() {
     return mapboxMapOptions;
   }
 
-  @Override
-  public boolean onOptionsItemSelected(MenuItem item) {
-    switch (item.getItemId()) {
-      case android.R.id.home:
-        // activity uses singleInstance for testing purposes
-        // code below provides a default navigation when using the app
```

Comment:
I believe this comment is still true?

Reply:
Yes, for activities that have a back button as home, this example has a hamburger instead.